packages/cozy-stack-client/src/FileCollection.spec.js

```diff
@@ -454,9 +519,58 @@ describe('FileCollection', () => {
       client.fetchJSON.mockClear()
     })
 
+    it('should not fetch json when file name is not valid', async () => {
+      try {
+        await collection.updateAttributes('42', {
+          dir_id: '123',
+          name: 'incorrect / name'
+        })
+      } catch (error) {
+        expect(client.fetchJSON.mock.calls.length).toEqual(0)
+      }
+    })
+
+    it('should throw error when file name contains illegal characters', async () => {
+      try {
+        await collection.updateAttributes('42', {
+          dir_id: '123',
+          name: 'incorrect / name'
+        })
+      } catch (error) {
+        expect(error.message).toEqual(
+          'Invalid filename containing illegal character(s): /'
+        )
+      }
+    })
+
+    it('should throw error when file name is forbidden', async () => {
+      try {
+        await collection.updateAttributes('42', {
+          dir_id: '123',
+          name: '.'
+        })
+      } catch (error) {
+        expect(error.message).toEqual(
+          'Invalid filename containing illegal character(s): .'
```

Comment:
Would it be possible to have a different error message here? `.` is a legal character but is an illegal name.